stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,74 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 50M
```

Comment:
This is 10 times the default value. I get that it reduces costs for customers to have fewer and larger API requests, but can we set the default to be something smaller and closer to the default? They can always change it themselves.

Reply:
The larger the request buffer the more likely the failures I think.

Reply:
These values are default as per this page: 
https://docs.fluentbit.io/manual/pipeline/outputs/s3/

Reply:
Updated to 5M